scripts/docker/documentation.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# This will set up an Astrobee melodic docker container using the non-NASA install instructions.
+# This image builds on top of the base melodic image building the code.
+# You must set the docker context to be the repository root directory
+
+ARG UBUNTU_VERSION=18.04
+FROM ubuntu:${UBUNTU_VERSION}
+
+# Install dependencies
+RUN apt-get update \
+  && apt-get install -y \
+  build-essential \
```

Comment:
Dockerfile style guideline: sort apt-get packages alphabetically (like Python isort)

Reply:
oh interesting, I was fully unaware of this, thank you!!